### PR TITLE
Lethabo mot patch 1

### PR DESCRIPTION
**Pull request by LethaboMot**
in project i:

-Added HTML and CSS
- Added images 
- Added README

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+    <title>Document</title>
+</head>
+
+<body>
+    <header id="toppart">
+        <nav class="toolbar">
+            <a href="">
+                <h2><img src="images/My Logo.svg"></h2>
+            </a>
```
Comment (leonardodiasb) [inline index.html:18]: - [ ] Your logo is correctly positioned and placed inside the header but it should not be wrapped in an `h2` tag. The `hx` tags are used for headings. Please remove the `h2` tag around the logo image.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+            <a href=""><img src="images/Hamburger button.svg" alt="three lines"></a>
+        </nav>
+    </header>
+
+    <section id="Introduction">
+        <h1>I'm Lethabo Glad to see you!</h1>
+        <div class="mid"><img src="images/Primary Text2.svg"></div>
```
Comment (leonardodiasb) [inline index.html:25]: - [ ] The paragraph below the main title of the page should be written by yourself and not copy/paste as an image, so you can modify it and use the right words that describe you. Kindly remove this div and write the paragraph as the template using a `p` tag.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+
+        <a href="#"><img src="images/connect.svg"></a>
```
Comment (leonardodiasb) [inline index.html:27]: - [ ] The 'LET'S CONNECT' should be written by yourself like the paragraph above it and not an `img` tag. Please remove the image, write the 'LET'S CONNECT' and style it like the Figma template.

**Review by leonardodiasb — CHANGES_REQUESTED** [review]
# Required Changes ♻️ ⚠️ 🚦 

Hi @LethaboMot, 

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!

mario_ladders

## Good points 👍 

 - No linter errors. ✔️
 - Correct use of Github flow. ✔️ 

## Aspects to improve ♻️

- [x] Check the comments under the review.
- [ ] It seems that you removed or forgot to commit the `images/` folder to this repository. None of the images in your HTML are loaded and this is making your app not look professional and not look similar to the Figma template. Kindly add the images you use in your project to this repository.
- Your design:
- 
image

 - Figma template:
 - 
image


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 4]
```diff
@@ -1,2 +1,31 @@
...
+## Getting Started
+To get a local copy up and running follow these simple example steps.
+### Prerequisites
+### Setup
+Clone this repository by using the command line :
+[`git clone https://github.com/LethaboMot/PORT.git`
+### Install
+### Usage
+### Run tests
+### Deployment
```
Comment (codecaiine) [inline README.md:17]: - [ ] [OPTIONAL] Please remove this empty to your readme file in order to make it more professional :smile:

**Review by codecaiine — CHANGES_REQUESTED** [review]
Hi @LethaboMot , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights
- Good project implementation :heavy_check_mark: 
- No linters errors :heavy_check_mark: 

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
- [ ] [OPTIONAL] Please make sure that your PR title is descriptive and tells what you are doing in your pull request in order to make your pr more professional. Example: Portfolio setup and mobile first  instead of  Lethabo mot patch 1 :+1:  

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me (@codecaiine) in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 5]
```diff
@@ -1,2 +1,31 @@
-# PORT
+Here’s what I did for the README doc:
+7:31
+![](https://img.shields.io/badge/Microverse-blueviolet)
+# Hello Microverse! project
+> This project shows Hello Microverse text in the webpage.
```
Comment (ha-manel) [inline README.md:5]: [Optional]

- [ ] Kindly update this section, this is for the "Hello Microverse" project while this is the "Portfolio" project 👍

File: README.md [hunk 6]
```diff
@@ -1,2 +1,31 @@
...
+To get a local copy up and running follow these simple example steps.
+### Prerequisites
+### Setup
+Clone this repository by using the command line :
+[`git clone https://github.com/LethaboMot/PORT.git`
+### Install
+### Usage
+### Run tests
+### Deployment
+## Authors
```
Comment (ha-manel) [inline README.md:18]: [Optional]

- [ ] It's better if you fill out these sections with information appropriate to this project 👍

**Review by ha-manel — CHANGES_REQUESTED** [review]
# STATUS: REQUIRED CHANGES ♻️

## Hi @LethaboMot 👋 
Great job so far, There are some changes that need to be implemented in order to proceed to the next project 👍  



## Highlights:  
* All tests passed ✔️

## Required Changes:
_Kindly check the comments below_

## Optional:
- [ ] It's better if you write the commit message as capitalized sentences in the imperative form (ex: "Add linters" and not "added linters") 👍 

_Kindly check the comments below_

------
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by frank1738 — APPROVED** [review]
Hi @LethaboMot , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


## Highlights

✅ Correct use of Github workflow
✅ No linter error
✅ README is okay

## Optional suggestions

N/A

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by emmiiorji — COMMENTED** [review]
Hi @LethaboMot, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a pull request that has been merged for a new milestone. Kindly note that at Microverse, every milestone is expected to be submitted in a new pull request.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by LethaboMot** [comment]
how can i submit to the the finish portfolio

**Review by christianbiring1 — COMMENTED** [review]
Hi @LethaboMot , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
# STATUS: APPROVED :tada: 
Iapproveokay

## Highlights
- [ ] `All suggestion by previous reviewers are completed ` :+1: 
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**